fiduswriter/document/static/js/modules/editor/index.js

```diff
@@ -340,6 +341,18 @@ export class Editor {
                     case 'patch_error':
                         showSystemMessage(gettext('Your document was out of sync and has been reset.'))
                         break
+                    case 'access_right':
+                        if (data.access_right !== this.docInfo.access_rights) {
+                            if (sendableSteps(this.view.state) && !(WRITE_ROLES).includes(data.access_right)) {
+                                // If the user's new rights does not allow him to update document , then download a copy of the
+                                // same and ask him to re-open the document.
+                                this.handleAccessRightModification()
+                            } else {
+                                addAlert('info', gettext(`Your Access rights have been modified. You now have ${data.access_right} access to this document.`))
```

Comment:
This needs to be written differently, as gettext() can only take simple strings. Therefore we need to use the `interpolate` function, like this:

```js
interpolate(
                gettext(
                    'Your Access rights have been modified. You now have %(accessRight)s access to this document.'
                ),
                {accessRights: data.access_right},
                true
            )
```

You can read more about interpolate here: https://docs.djangoproject.com/en/3.1/topics/i18n/translation/#interpolate

Reply:
Hi @johanneswilm , I've updated this message to use the interpolate function like you suggested.